### ISSUE TITLE
Option to limit player to spectator mode until account is linked

### ISSUE THREAD

**Issue by Freddo3000** [DiscordSRV/DiscordSRV#877]
This is already doable to some extent via roles and permissions, though it would be useful to simply have the option available as a simple toggle for servers running without permission plugins.

Preferably also with a prompt to link their account and a discord link.

**Comment by Scarsz** [comment]
Good idea though it's use would be somewhat limited as servers that require linked accounts usually have parts of the server that are only for linked members to see (if using subscriber roles, etc.)

Could definitely be useful for a streamer/event server though
